src/main/java/backend/resource/TurboIssue.java

```diff
@@ -438,13 +443,15 @@ public void addLabel(TurboLabel label) {
     public Optional<Integer> getMilestone() {
         return milestone;
     }
-
-    public void setMilestone(Integer milestone) {
-        this.milestone = Optional.of(milestone);
+    
+    public void setMilestoneById(Integer milestone) {
+        this.milestone = (milestone == null) ? Optional.empty() : Optional.of(milestone);
```

Comment:
Not too sure what you mean... A null milestone is a valid input, when the user wishes to demilestone an issue.